sailthru_content/services/sailthru_translation_service.py

```diff
@@ -133,6 +143,10 @@ def translate_course_run(self, course_run, course, program_dictionary=None):
         if len(tags) > 0:
             sailthru_content['tags'] = ", ".join(tags)
 
+        seats = course_run.get('seats')
+        if seats:
```

Comment:
`seats` is a empty list `[ ]` and will evaluates `True` only when there is at least one `seat` type exists.